### PR TITLE
lowercase words and rearrange pages

### PR DESCRIPTION
**Pull request by iregina**
Completed all the easy changes in the navigation bar

### CODE REVIEW

File: src/components/DocsNavs.re [hunk 1]
```diff
@@ -472,18 +472,18 @@ module SideNav = {
...
-            title={intl->Intl.formatMessage(zkAppsForEthereumDevelopers)}
-            slug="zkapps-for-ethereum-developers"
-          />
-          <Item
-            title={intl->Intl.formatMessage(simpleAnonymousMessageBoardTutorial)}
-            slug="simple-anonymous-message-board-tutorial"
-          />
           <Item
           title={intl->Intl.formatMessage(advancedConcepts)}
           slug="advanced-concepts"
```
Comment (jasongitmail) [inline src/components/DocsNavs.re:477]: `advanced-snarkyjs`. And any links to it too

File: src/components/DocsNavs.re [hunk 2]
```diff
@@ -472,18 +472,18 @@ module SideNav = {
...
-          <Item
-            title={intl->Intl.formatMessage(zkAppsForEthereumDevelopers)}
-            slug="zkapps-for-ethereum-developers"
-          />
-          <Item
-            title={intl->Intl.formatMessage(simpleAnonymousMessageBoardTutorial)}
-            slug="simple-anonymous-message-board-tutorial"
-          />
           <Item
           title={intl->Intl.formatMessage(advancedConcepts)}
```
Comment (jasongitmail) [inline src/components/DocsNavs.re:476]: `advancedSnarkyjs` or `advancedSnarkyJs`

File: src/components/DocsNavs.re [hunk 3]
```diff
@@ -762,18 +762,18 @@ module Dropdown = {
...
           slug="advanced-concepts"
```
Comment (jasongitmail) [inline src/components/DocsNavs.re:767]: Should be deleted? I don't see 2x on the UI, so not sure

**Review by jasongitmail — COMMENTED** [review]
We should change the H1 title on that page to `Advanced SnarkyJS` too

File: src/components/DocsNavs.re (hunk 1 above)
Comment (iregina) [reply src/components/DocsNavs.re:477]: Done

File: src/components/DocsNavs.re (hunk 2 above)
Comment (iregina) [reply src/components/DocsNavs.re]: Done

File: src/components/DocsNavs.re (hunk 3 above)
Comment (iregina) [reply src/components/DocsNavs.re]: Not deleted but changed to snarkyjs

**Comment by jasongitmail** [comment]
@iregina Branch has conflicts. You might need to rebase and open a new PR.

**Comment by jasongitmail** [comment]
Regina opened a new PR https://github.com/MinaProtocol/docs/pull/212

**Pull request closed without merging**